src/main/scala/dpla/ingestion3/executors/EnrichExecutor.scala

```diff
@@ -124,25 +123,20 @@ trait EnrichExecutor extends Serializable {
     val placeMessages = messages.filter("field='place'")
 
     val logEnrichedFields = List(
-      "all" -> messages,
       "type" -> typeMessages,
       "date" -> dateMessages,
       "language" -> langMessages,
       "place" -> placeMessages
-    )
-      .filter { case (_, data: Dataset[_]) => data.count() > 0 }
+    ).filter { case (_, data: Dataset[_]) => data.count() > 0 } // drop empty
```

Comment:
This could end up rerunning the calculations that build these datasets more than once. We should probably do an audit of the execution plan of this code at some point to see if it's doing redundant work.

Reply:
agree